packages/lodestar/src/node/notifier.ts

```diff
@@ -75,20 +80,21 @@ export async function runNodeNotifier({
             clockSlotRow,
             headRow,
             finalizedCheckpointRow,
+            ...backfilledStat,
```

Comment:
I won't add any info to the notifier. Users don't really care about backfill status. We must ensure it works and they trust us that it does. We should use metrics and API debug endpoints to learn about the status of backfill sync